promql/parser/lex.go

```diff
@@ -617,6 +617,16 @@ func lexBuckets(l *Lexer) stateFn {
 		l.bracketOpen = false
 		l.emit(RIGHT_BRACKET)
 		return lexHistogram
+	case isAlpha(r):
```

Comment:
Could you add a test case for this to lex_test.go?

Reply:
added!